### ISSUE TITLE
Deprecate third (3rd) party repository in favour of flatpaks

### ISSUE THREAD

**Issue by ermo** [getsolus/packages#1341]
Depends on:
- getsolus/packages#716
- getsolus/packages#718
- getsolus/packages#1316

*David edit*
Related:

- getsolus/3rd-party#72
- getsolus/3rd-party#73
- getsolus/3rd-party#74

- getsolus/solus-sc/pull/315

**Comment by davidjharder** [comment]
Related https://github.com/getsolus/help-center-docs/issues/460

**Comment by brent111** [comment]
Microsoft Core Fonts
```
sudo eopkg bi --ignore-safety https://raw.githubusercontent.com/getsolus/3rd-party/master/desktop/font/mscorefonts/pspec.xml
sudo eopkg it mscorefonts*.eopkg;sudo rm mscorefonts*.eopkg
```

I know I'm getting ahead of your deprecation schedule, whatever that may be, but I use (need for work) and install mscorefont  package in every solus installation I do.
MSCorefonts are not available in Flatpak, or at lease I couldn't find them.
Trying to help; this is not a complaint or request. Thanks.

**Comment by Justinzobel** [comment]
@brent111 grab the pspec file, and you can easily update it locally and build them whenever they get updates. If you have any issues updating them, jump in the Packaging channel #solus-packaging:matrix.org and ask.

**Comment by brent111** [comment]
> @brent111 grab the pspec file, and you can easily update it locally and build them whenever they get updates. If you have any issues updating them, jump in the Packaging channel #solus-packaging:matrix.org and ask.

appreciate the pspec info, never tried it. good to see you back in the community.

**Comment by Justinzobel** [comment]
Yeah the pspec stuff is far from magic. Once Serpent and Solus merge I think it'll be even easier as I've heard Ikey talk about having source packages like these are that can be easily built by users.

**Comment by TraceyC77** [comment]
Jetbrains - has an official installer, which is recommended over the flatpak, which is horrible. We are reaching out to see if they will let us package this instead.
